vars/withAcrClient.groovy

```diff
@@ -1,11 +1,11 @@
 def call(String subscription, Closure block) {
-  withDocker('hmcts/cnp-aks-client:az-2.3.0-kubectl-1.18.0-helm-3.1.2', null) {
+//  withDocker('hmcts/cnp-aks-client:az-2.3.0-kubectl-1.18.0-helm-3.1.2', null) {
```

Comment:
so we need to add these to the VM agent image as well then?

Reply:
I'd rather not if possible...